app/build.gradle

```diff
@@ -117,7 +117,7 @@ task zipalignRelease << {
 
 release {
     tagTemplate = 'v${version}'
-    buildTasks = ['assembleRelease']
+    buildTasks = ['build']
```

Comment:
@pazaan this seemed strange/suspicious, I presume it works for you? I can see that `assembleRelease` task in the Gradle panel in the IDE. Without this change attempting to even init Gradle when the project is open fails unable to find the task.